go.mod

```diff
@@ -0,0 +1,14 @@
+module github.com/myzhan/boomer
+
+go 1.14
```

Comment:
This PR is based off of the other one that adds it. Obviously I'd rebase if you don't want the other change, doing it this way was easiest way to prevent merge conflicts when I built the separate PRs.